webclient/src/components/KnownHosts/KnownHosts.tsx

```diff
@@ -152,20 +193,31 @@ const KnownHosts = (props) => {
     );
   };
 
+  const testConnection = () => {
+    setTestingConnection(s => ({
+      testing: true,
```

Comment:
Would an ENUM be better compared to three unique booleans? Is that something TS can support?

Reply:
Not really, we'd then have to do value comparisons down where we use these booleans.